config/stylelint.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  "rules": {
+    "function-url-quotes": "single",
```

Comment:
@rustemginiyatullin Yeah, it was just the test configuration to check the tool is working)